src/components/shift-schedule/ScheduleTable.tsx

```diff
@@ -0,0 +1,160 @@
+import { ScheduleCell } from '@/components/shift-schedule/ScheduleCell';
+import {
+  Table,
+  TableBody,
+  TableCaption,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from '@/components/ui/Table';
+import { useFormatter, useTranslations } from 'next-intl';
+
+type ScheduleEntryProps = {
+  members: {
+    name: string;
+  }[];
+};
+
+type ScheduleDayProps = {
+  first: ScheduleEntryProps;
+  second: ScheduleEntryProps;
+  third: ScheduleEntryProps;
+  fourth: ScheduleEntryProps;
+};
+
+type ScheduleTableProps = {
+  week: {
+    monday: ScheduleDayProps;
+    tuesday: ScheduleDayProps;
+    wednesday: ScheduleDayProps;
+    thursday: ScheduleDayProps;
+    friday: ScheduleDayProps;
+  };
+};
+
+function ScheduleTable({ week }: ScheduleTableProps) {
+  const t = useTranslations('shiftSchedule.scheduleTable');
+  const format = useFormatter();
+
+  const days = [
+    'monday',
+    'tuesday',
+    'wednesday',
+    'thursday',
+    'friday',
+  ] as const;
+  const timeslots = ['first', 'second', 'third', 'fourth'] as const;
+
+  function getDateTimeRange(timeslot: string) {
+    let firstDate: Date;
+    let secondDate: Date;
+
+    switch (timeslot) {
+      case timeslots[0]:
+        firstDate = new Date(0, 0, 0, 10, 15, 0, 0);
+        secondDate = new Date(0, 0, 0, 12, 7, 0, 0);
+        break;
+
+      case timeslots[1]:
+        firstDate = new Date(0, 0, 0, 12, 7, 0, 0);
+        secondDate = new Date(0, 0, 0, 14, 7, 0, 0);
+        break;
+
+      case timeslots[2]:
+        firstDate = new Date(0, 0, 0, 14, 7, 0, 0);
+        secondDate = new Date(0, 0, 0, 16, 7, 0, 0);
+        break;
+
+      case timeslots[3]:
+        firstDate = new Date(0, 0, 0, 16, 7, 0, 0);
+        secondDate = new Date(0, 0, 0, 18, 0, 0, 0);
+        break;
+
+      default:
+        firstDate = new Date();
+        secondDate = new Date();
+    }
+
+    return format.dateTimeRange(firstDate, secondDate, {
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+    });
+  }
+
+  return (
+    <>
+      {/* Table shown on small screens */}
+      <div className='sm:hidden'>
+        {days.map((day, index) => (
+          <Table key={day} className={index === 0 ? '' : 'mt-8'}>
```

Comment:
```suggestion
      <div className='mt-16 sm:hidden'>
        {days.map((day) => (
          <Table key={day}>
```
This will achieve the same result. The administrator menu above already has a margin of 8, but we want to push the table even further.

Reply:
The point of this part is to add margin to all elements except the first one. This suggestion would add margin to their container, basically the same as giving margin only to the first one. Also, first: doesn't work here for some reason, which is why I'm using the index.